### ISSUE TITLE
Make header buttons more accessibile

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2416]
Originally reported on Google Code with ID 2151

```
Hi,

My apologies if this has been reported already. I did some due diligence but is quite
likely I may not have searched correctly. It pertains to the accessibility of this
great plugin.

Step 1
------
Using demo on fullCalendar site [1]. The only code added was via Firebug to highlight
elements receiving focus.

Step 2
-----
Skipping - there is no bug specifically demonstrable via a jsFiddle, so skipping straight
to:

Step 3
-----

Please see screencast [2]

1. Observe that using the keyboard (as most screen readers will, and a certain number
of mobility impaired users will as well) all of the elements save one in the markup
produced by fullCalendar never receives a focus. They will be preceivable (but not
as controls since they are spans, divs and table cells), but inaccessible.

Users will be unable to act on:

a. Navigation controls (Today, Next / Prev, month, week day)
b. Event details. These are unavailable - eventClick cannot be triggered because the
events cannot receive keyboard focus
c. Creating new events. Table cells that are the anchors for the eventClick trigger
are also not operable via the keyboard, since they cannot receive focus either

b and c where also verified on a local installation. A standards based approach might
be to decorate the DOM with ARIA roles that would identify these components as having
the role of controls, managing the focus (and tab order) and managing their state as
well.

Thanks! If this issue has been addressed or represents a misunderstanding do let me
know.

[1] http://arshaw.com/fullcalendar/
[2] http://screencast.com/t/fsnaMiLl89jR

```

Reported by `gsilver@umich.edu` on 2014-05-14 17:19:12
